app/services/comment_notifier.rb

```diff
@@ -4,18 +4,18 @@ def initialize(broadcaster: PusherBroadcaster.new, serializer: CommentSerializer
     @serializer = serializer
   end
 
-  def send_comment(comment, channel)
+  def send_comment(comment:, channel:, event: 'comment_added')
```

Comment:
maybe we should create `comment_added` and `comment_rejected` method which will know what event should be pushed